### PR TITLE
Add missing scheme_test.go

### PR DESCRIPTION
**Pull request by lianghao208**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Adds missing scheme_test.go to ensure plugin arguments are decoded as expected.
 
 **Which issue(s) this PR fixes**:

#236

### CODE REVIEW

File: pkg/apis/config/scheme/scheme_test.go [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	v1 "k8s.io/kube-scheduler/config/v1"
+	defaultconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/config"
```
Comment (Huang-Wei) [inline pkg/apis/config/scheme/scheme_test.go:30]: nit: add an empty line above

File: pkg/apis/config/scheme/scheme_test.go [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+// into their appropriate internal types and defaults are applied.
+func TestCodecsDecodePluginConfig(t *testing.T) {
+	testCPUQuantity, _ := resource.ParseQuantity("1000m")
+	testCases := []struct {
+		name         string
+		data         []byte
+		wantErr      string
+		wantProfiles []defaultconfig.KubeSchedulerProfile
+	}{
+		//v1beta1 tests
```
Comment (Huang-Wei) [inline pkg/apis/config/scheme/scheme_test.go:51]:
```suggestion
		// v1beta1 tests
```

File: pkg/apis/config/scheme/scheme_test.go [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+									Type:    config.Prometheus,
+									Address: "http://prometheus-k8s.monitoring.svc.cluster.local:9090",
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
```
Comment (Huang-Wei) [inline pkg/apis/config/scheme/scheme_test.go:143]: Can you add more cases?

- leave some fields unspecified, and then verify they get defaulted properly
- provide some illegal args, and verify if we get the validation error

Comment (lianghao208) [reply pkg/apis/config/scheme/scheme_test.go:230]: done, thanks for the review!

**Pull request merged**